### ISSUE TITLE
Image Selector - keyboard navigation doesn't work in flat mode

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#988]
1. Open new Image Selector wizard,
2. Type a  part of the existing image-name .
![image](https://user-images.githubusercontent.com/3728712/65149511-e5c6bf80-da2a-11e9-8ab0-a5223bdb1bb8.png)

3. Try select an item from a combobox with the up/down left/right arrow keys.

BUG:  left/right arrow keys do not  navigate through options.
         up/down keys select only the first option

**Comment by ashklianko** [comment]
Issue moved to [enonic/lib-admin-ui #1138](https://github.com/enonic/lib-admin-ui/issues/1138) via [**ZenHub**](https://www.zenhub.com/)
